### ISSUE TITLE
[Accessibility]A11y_VSCode_ShowOpenedEditors_ColorContrast:Color contrast ratio is less than 4.5:1 for the placeholder and the list items present on the screen 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52604]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1)Launch VS Code.
Navigate to welcome screen and beside opened editors pane at top(in middle pane)->Select More(…)->Select "Show Opened Editors" from dropdown.
2.Navigate to editor and search for any commands.
3)Open Color contrast analyser and check the ratio.

**Actual:**
Color contrast ratio is less than 4.5:1 for the placeholder tex and the list items present on the screen.

Same issue has been observed in all the light and dark themes

**Expected:**
Color Contrast ratio should not be less than 4.5:1.
Same issue observed on all other Color theme .

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not be able to see the controls.

**MAS Reference:**
MAS1.4.3-https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment for Reference:**
![accessibility a11y_vscode_showopenededitors_colorcontrast](https://user-images.githubusercontent.com/39204909/41718918-0bccdaee-757c-11e8-8d09-e5af84992800.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51831
